### PR TITLE
Update sln, added XUnit tests

### PR DESCRIPTION
**Pull request by VasiliiUsupov**
Add XUnit tests

### CODE REVIEW

File: WebHDFS.Kitty.XUnitTests/DataTestUtilityControl.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace WebHDFS.Kitty.XUnitTests
+{
+    public sealed class DataTestUtilityControl : FactAttribute
```
Comment (galakt) [inline WebHDFS.Kitty.XUnitTests/DataTestUtilityControl.cs]: Rename to `CheckConnStrSetupFact`

**Comment by galakt** [comment]
.cache files should be excluded

File: WebHDFS.Kitty.sln [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29509.3
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebHDFS.Kitty", "WebHDFS.Kitty\WebHDFS.Kitty.csproj", "{90E44E8F-254A-48BA-9387-2F359F6388AE}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebHDFS.Kitty.IntegrationTests", "WebHDFS.Kitty.IntegrationTests\WebHDFS.Kitty.IntegrationTests.csproj", "{ED88D91D-2F8C-40FE-B552-6558E4655E5F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebHDFS.Kitty.XUnitTests", "WebHDFS.Kitty.XUnitTests\WebHDFS.Kitty.XUnitTests.csproj", "{14D67D2E-D412-4515-B130-180BCE987178}"
```
Comment (galakt) [inline WebHDFS.Kitty.sln]: `WebHDFS.Kitty.IntegrationTests` project should be removed, `WebHDFS.Kitty.XUnitTests` project and folder should be renamed to `WebHDFS.Kitty.IntegrationTests`

**Comment by VasiliiUsupov** [comment]
Delete old IntegrationTests, rename XUnitTests, delete .cache

File: WebHDFS.Kitty.IntegrationTests/WebHDFS.Kitty.IntegrationTests.csproj [hunk 3]
```diff
@@ -1,26 +1,20 @@
...
-    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\WebHDFS.Kitty\WebHDFS.Kitty.csproj" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <None Update="Samples\SampleTextFile.txt">
```
Comment (galakt) [inline WebHDFS.Kitty.IntegrationTests/WebHDFS.Kitty.IntegrationTests.csproj:21]: Sample file is missed

File: WebHDFS.Kitty.IntegrationTests/CheckConnStrSetupFact.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+using System.Text;
+using Xunit;
+
+namespace WebHDFS.Kitty.IntegrationTests
+{
+    public sealed class CheckConnStrSetupFact : FactAttribute
+    {
+        public CheckConnStrSetupFact()
+        {
+            if (DataTestUtility.HdfsConnStr == null)
```
Comment (galakt) [inline WebHDFS.Kitty.IntegrationTests/CheckConnStrSetupFact.cs]: Change to string.IsNullOrWhiteSpace

**Comment by VasiliiUsupov** [comment]
Add changes with txt file and Change to string.IsNullOrWhiteSpace

**Pull request merged**